### PR TITLE
added learning goals React and corrected link js1

### PR DESCRIPTION
**Pull request by mkruijt**

### CODE REVIEW

**Review by jasongwartz — APPROVED** [review]
I have no idea if these are real React terms or not, but I have faith.

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 - Write basic shell scripts to ease the programming life.
 
 
-## [JavaScript1](https://github.com/HackYourFuture/JavaScript/)/[GIT](https://github.com/HackYourFuture/Git)/ [debugging](https://github.com/HackYourFuture/debugging)
+## [JavaScript1](https://github.com/HackYourFuture/JavaScript1/)/[GIT](https://github.com/HackYourFuture/Git)/ [debugging](https://github.com/HackYourFuture/debugging)
```
Comment (remarcmij) [inline README.md:42]: Are we actually still planning to do separate debugging hangouts? Is there any conclusion of the first attempts?

**Comment by mkruijt** [comment]
You faith in me is highly appreciated @jasongwartz 💃

File: README.md (hunk 1 above)
Comment (mkruijt) [reply README.md:42]: The idea is to see if we can incorporate it in the javascript sessions, let's talk about it Sunday!

**Pull request merged**